src/frontend/components/DashboardVideoLiveControlPane/widgets/DashboardVideoLiveWidgetThumbnail/ThumbnailDisplayer/index.tsx

```diff
@@ -0,0 +1,47 @@
+import { Image } from 'grommet';
+import React from 'react';
+import { defineMessages, useIntl } from 'react-intl';
+
+import { Thumbnail, videoSize } from 'types/tracks';
+import { videoSizeMapping } from 'utils/videoSizeMapping';
+
+interface ThumbnailDisplayerProps {
+  rounded?: boolean;
+  thumbnail: Thumbnail;
+}
+
+const messages = defineMessages({
+  thumbnailAlt: {
+    defaultMessage: 'Live video thumbnail',
+    description:
+      'Accessibility text for the live video thumbnail in the Dashboard.',
+    id: 'components.ThumbnailDisplayer.thumbnailAlt',
+  },
+});
+
+export const ThumbnailDisplayer = ({
+  rounded,
+  thumbnail,
```

Comment:
I don't think you should use a thumbnail here, you are using only the urls.
If you change your component props to receive urls you will be able to use this component to render the default background with it.
CF : https://github.com/openfun/marsha/pull/1568/files#r891150092

Reply:
I put everything in one component